foreman/data_refinery_foreman/surveyor/array_express.py

```diff
@@ -129,27 +125,30 @@ def create_experiment_from_api(self, experiment_accession_code: str) -> (Experim
                          experiment_accession_code=experiment_accession_code,
                          survey_job=self.survey_job.id)
         except Experiment.DoesNotExist:
+            # We aren't sure these fields will be populated, or how many there will be.
+            # Try to join them all together, or set a sensible default.
+            experiment_descripton = ""
+            if "description" in parsed_json and len(parsed_json["description"]) > 0:
+                for description_item in parsed_json["description"]:
+                    if "text" in description_item:
+                        experiment_descripton = experiment_descripton + description_item["text"] + "\n"
+
+            if experiment_descripton == "":
+                experiment_descripton = "Description not available.\n"
```

Comment:
Why the newline?

Reply:
Honestly, I don't remember. This was pulled in via the collab review so I guess we decided this is how we should do it.

I'm guessing the reason is that if `experiment_description` does actually get populated we join all the lines of it together into a string with a newline inbetween each one (see line 134) so it would end in a newline so this is consistent with that.